src/diagonal.works/b6/ingest/features.go

```diff
@@ -509,6 +529,19 @@ func (a *AreaFeature) FillFromOSMWay(way *osm.Way) {
 	a.AreaMembers.FillFromOSMWay(way)
 }
 
+func (a *AreaFeature) References() []b6.Reference {
```

Comment:
I quite like this idea of having a feature be responsible for returning its references